godot-core/src/obj/onready.rs

```diff
@@ -36,21 +40,27 @@ use std::mem;
 /// [option]: std::option::Option
 /// [lazy]: https://docs.rs/once_cell/1/once_cell/unsync/struct.Lazy.html
 ///
-/// # Example
+/// # Requirements
+/// - The class must have an explicit `Base` field (i.e. `base: Base<Node>`).
+/// - The class must implement `Inherits<Node>` (otherwise `ready()` would never be called anyway).
```

Comment:
There is actually not even a `ready()` method for other classes
```suggestion
/// - The class must inherit `Node` (otherwise `ready()` would not exist anyway).
```